runtime/doc/lsp.txt

```diff
@@ -173,129 +162,72 @@ when creating a new client. Keys are LSP method names: >vim
                                                                   *lsp-method*
 
 Methods are the names of requests and notifications as defined by the LSP
-specification. These LSP requests/notifications are defined by default:
-
-    callHierarchy/incomingCalls
-    callHierarchy/outgoingCalls
-    textDocument/codeAction
-    textDocument/completion
-    textDocument/declaration*
-    textDocument/definition
-    textDocument/documentHighlight
-    textDocument/documentSymbol
-    textDocument/formatting
-    textDocument/hover
-    textDocument/implementation*
-    textDocument/inlayHint
-    textDocument/publishDiagnostics
-    textDocument/rangeFormatting
-    textDocument/references
-    textDocument/rename
-    textDocument/semanticTokens/full
-    textDocument/semanticTokens/full/delta
-    textDocument/signatureHelp
-    textDocument/typeDefinition*
-    window/logMessage
-    window/showMessage
-    window/showDocument
-    window/showMessageRequest
-    workspace/applyEdit
-    workspace/inlayHint/refresh
-    workspace/symbol
-
-* NOTE: These are sometimes not implemented by servers.
+specification. These LSP requests/notifications are defined by default in the
+Nvim LSP client (but depends on server support):
+
+- callHierarchy/incomingCalls
+- callHierarchy/outgoingCalls
```

Comment:
Now we can generate `runtime/doc/lsp.txt` as a "flow" layout instead of fixed-width.